src/solidity-frontend/solidity_convert.h

```diff
@@ -0,0 +1,113 @@
+#ifndef SOLIDITY_FRONTEND_SOLIDITY_CONVERT_H_
+#define SOLIDITY_FRONTEND_SOLIDITY_CONVERT_H_
+
+#include <memory>
+#include <iostream>
```

Comment:
Don´t use iostream, you should use the message interface.

Reply:
Done. 
Removed. The code is now using the msg and fmt interfaces.